src/custom-element.js

```diff
@@ -1716,6 +1730,27 @@ function createBaseCustomElementClass(win) {
       });
     }
 
+    insertSandboxAnalytics_() {
+      this.sandboxAnalyticsConfigPromise_ =
+          this.implementation_.createSandboxAnalyticsConfigCallback();
+      if (this.sandboxAnalyticsConfigPromise_) {
+        this.sandboxAnalyticsConfigPromise_.then(config => {
+          this.sandboxAnalyticsConfigPromise_ = null;
+          this.sandboxAnalyticsElement_ = insertAnalyticsElement(
```

Comment:
This is a part of core. Using `insertAnalyticsElement` would to the core binary.